dist/leaflet.css

```diff
@@ -557,6 +612,30 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border: 1px solid #999;
 	}
 
+/* Dark Mode */
+.leaflet-darkmode .leaflet-popup-content-wrapper,
+.leaflet-darkmode .leaflet-popup-tip {
+	background: #333;
+	color: #fff;
+	box-shadow: 0 3px 14px rgba(0,0,0,0.4);
+	}
+.leaflet-darkmode.leaflet-container a.leaflet-popup-close-button{
+	color: #ddd;
+	}
+.leaflet-darkmode.leaflet-container a.leaflet-popup-close-button:hover {
+	color: #999;
+	}
+.leaflet-darkmode .leaflet-popup-scrolled {
+	border-bottom: 1px solid #ddd;
+	border-top: 1px solid #ddd;
+	}
+.leaflet-darkmode.leaflet-oldie .leaflet-control-zoom,
+.leaflet-darkmode.leaflet-oldie .leaflet-control-layers,
+.leaflet-darkmode.leaflet-oldie .leaflet-popup-content-wrapper,
+.leaflet-darkmode.leaflet-oldie .leaflet-popup-tip {
+	border: 1px solid #999;
+	}

```

Comment:
I think we're doing away with `leaflet-oldie` soon, should we exclude it from this PR?

Reply:
`leaflet-oldie` was removed in https://github.com/Leaflet/Leaflet/pull/8559.